.eslintrc.cjs

```diff
@@ -35,6 +35,20 @@ module.exports = {
       'rules': {
         '@typescript-eslint/no-non-null-assertion': 2,
         '@typescript-eslint/no-explicit-any': 2,
+        '@typescript-eslint/member-delimiter-style': ['error', {
+          'multiline': {
+            'delimiter': 'semi',
+            'requireLast': true,
+          },
+          'singleline': {
+            'delimiter': 'semi',
+            'requireLast': true,
+          },
+        }],
+        'object-shorthand': 'error',
+        '@typescript-eslint/method-signature-style': ['error', 'method'],
+        '@typescript-eslint/explicit-module-boundary-types': ['error'],
```

Comment:
This seems good, especially with `allowDirectConstAssertionInArrowFunctions: true` as a default.

Do you think `allowHigherOrderFunctions: false` is worth setting?

Reply:
> Do you think `allowHigherOrderFunctions: false` is worth setting?

Good idea.